mtp_cashbook/templates/cashbook/transaction_batch_list.html

```diff
@@ -106,22 +105,14 @@ <h1 class="error-summary-heading heading-medium" id="error-summary-heading">{% t
 
     <dialog id="incomplete-batch-dialog" class="Dialog u-nojs-hidden" data-hide-close="true" data-disable-backdrop-close="true">
       <div class="Dialog-inner">
-        <h3>{% trans "Confirm submission" %}</h3>
-        <p>{% trans "Are you sure you want to submit only the selected payments as a batch?" %}</p>
-        <button type="submit" class="button" value="override">{% trans "Yes, submit" %}</button>
-        <button type="button" class="button button-secondary js-Dialog-close">{% trans "No, continue editing" %}</button>
+        <h3>{% trans "Are you sure?" %}</h3>
+        <p>{% trans "Do you want to submit only the selected credits?" %}</p>
+        <button type="submit" class="button" value="override">{% trans "Yes" %}</button>
```

Comment:
Would it be worth making the `Yes` button explain what it does? Something like `Submit selected credits`, perhaps, @leighmoney?